tekton/pipeline-templates/params/common.yml.j2

```diff
@@ -0,0 +1,81 @@
+# CLI Version
```

Comment:
Bad merge possibly?  `cli_version` is no more after the last round updates to optimize development flow.

Reply:
Same reason of the "fvt" parameters described below... `cli_version` is used inside run-pipeline task to download the right version of pipelineruns from artifactory.

Reply:
Makes sense ... Should move/set that param specifically for that task now, rather than have it in the global/shared param definition.